src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -74,27 +74,47 @@ const TitleBarWarning = (messageComponents) => {
   });
 };
 
+const TitleBarSpinner = (messageComponents) => {
+  return h(TitleBar, {
+    title: div({ role: 'alert', style: { display: 'flex', alignItems: 'center' } }, [
+      spinner({
+        size: 64,
+        style: {
+          position: 'relative',
+          display: 'flex',
+          alignItems: 'center',
+          backgroundColor: colors.warning(0.1),
+          padding: '1rem',
+          borderRadius: '0.5rem',
+        },
+      }),
+      span({ style: { color: colors.dark(), fontSize: 14 } }, messageComponents),
+    ]),
+    style: { backgroundColor: colors.warning(0.1), borderBottom: `1px solid ${colors.warning()}` },
+  });
+};
+
 const AzureWarning = () => {
   const warningMessage = [
     'Do not store Unclassified Confidential Information in this platform, as it violates US Federal Policy (ie FISMA, FIPS-199, etc) unless explicitly authorized by the dataset manager or governed by your own agreements.',
   ];
   return TitleBarWarning(warningMessage);
 };
 
-const GooglePermissionsWarning = () => {
+const GooglePermissionsSpinner = () => {
   const warningMessage = [
-    'Google is syncing permissions for this workspace, which may take a few minutes or longer. During this time, access to workspace features will be unavailable. ',
+    'Terra synchronizing permissions with Google. This may take a couple moments.',
     h(
       Link,
       {
         href: 'https://support.terra.bio/hc/en-us/community/posts/12380560785819-Delays-in-Google-IAM-permissions-propagating',
         ...Utils.newTabLinkProps,
       },
-      [div(['Learn more here.'])]
+      [div(['Learn more here. ', icon('pop-out')])]
```

Comment:
Thanks for adding the icon! I would also suggest replacing "here" with something more meaningful, like "Learn more about permissions propagation.". This is because some people navigate by link, so it is good if the link text explains what it goes to.

Reply:
How about "permissions synchronization" instead of "permissions propagation". We're no longer waiting for propagation, we're waiting for Rawls to sync FastPass permissions for the user on the workspace.

Reply:
@tlangs although that makes me realize that the article we are linking to is also out-of-date. https://support.terra.bio/hc/en-us/community/posts/12380560785819-Delays-in-Google-IAM-permissions-propagating

Reply:
Do we still wish to link to that article at all? If so, perhaps a new comment can be added which explains the new state?

Reply:
I'm back from vacation! I'm comfortable just removing the link to the article, but I want some UX input on that.

Reply:
OK, if you don't need a link, you should be able to go back to the original blue color.